server/src/test/java/io/spine/examples/chatspn/server/ChatSpnServerTest.java

```diff
@@ -0,0 +1,219 @@
+    void entitySubscription() throws ExecutionException, InterruptedException,
+                                     TimeoutException {
+        var client = createClient();
+        var command = createAccount();
+        var futureEntity = new CompletableFuture<UserProfile>();
+        client.asGuest()
+              .subscribeTo(UserProfile.class)
+              .observe(futureEntity::complete)
+              .post();
+        client.asGuest()
+              .command(command)
+              .postAndForget();
+        var expectedUserProfile = userProfile(command);
+
+        assertExpectedFields(futureEntity.get(2, SECONDS), expectedUserProfile);
+    }
+
+    @Test
+    @DisplayName("subscribe to the `UserProfile` projection state by ID " +
+            "and call 'observing consumer' when the `UserProfile` projection with provided ID " +
+            "will be created in response to the `CreateAccount` command")
+    void entitySubscriptionById() throws ExecutionException, InterruptedException,
+                                         TimeoutException {
+        var client = createClient();
+        var firstCommand = createAccount();
+        var secondCommand = createAccount();
+        var futureEntity = new CompletableFuture<UserProfile>();
+        client.asGuest()
+              .subscribeTo(UserProfile.class)
+              .byId(secondCommand.getUser())
+              .observe(futureEntity::complete)
+              .post();
+        client.asGuest()
+              .command(firstCommand)
+              .postAndForget();
+        client.asGuest()
+              .command(secondCommand)
+              .postAndForget();
+        var expectedUserProfile = userProfile(secondCommand);
+
+        assertExpectedFields(futureEntity.get(2, SECONDS), expectedUserProfile);
+    }
+
+    @Test
+    @DisplayName("subscribe to the `AccountCreated` event and call 'observing consumer' " +
+            "when the `AccountCreated` event will be emitted " +
+            "in response to the `CreateAccount` command")
+    void eventSubscription() throws ExecutionException, InterruptedException,
+                                    TimeoutException {
+        var client = createClient();
+        var command = createAccount();
+        var futureEvent = new CompletableFuture<AccountCreated>();
+        client.asGuest()
+              .subscribeToEvent(AccountCreated.class)
+              .observe(futureEvent::complete)
+              .post();
+        client.asGuest()
+              .command(command)
+              .postAndForget();
+        var expectedEvent = accountCreated(command);
+
+        assertExpectedFields(futureEvent.get(2, SECONDS), expectedEvent);
+    }
+
+    @Test
+    @DisplayName("subscribe to the `AccountCreated` event by ID " +
+            "and call 'observing consumer' when the `AccountCreated` event with provided 'id' field " +
+            "will be emitted in response to the `CreateAccount` command")
```

Comment:
"is emitted".

It's here and in other similar cases.

Please have a read on conditionals in English.